eating_cookies/eating_cookies.py

```diff
@@ -8,11 +8,27 @@
 def eating_cookies(n, cache=None):
       
```

Comment:
ok, see if you can at least do a recursive way to do this, and not hard code it, think of the base case, maybe the first 3, and look for a pattern, hint: 1,1,2=4 for 3 ways to eat a cookie ............................ 1+2+4=7 for 4 ways to eat a cookie, im fairly confident there will be 13 ways to eat 5 cookies